README.md

```diff
@@ -1,11 +1,11 @@
 ![Logo](https://raw.githubusercontent.com/idealista/elasticsearch_role/master/logo.gif)
 
-[![Build Status](https://travis-ci.org/idealista/elasticsearch_role.png)](https://travis-ci.org/idealista/elasticsearch_role)
+[![Build Status](https://travis-ci.com/idealista/elasticsearch_role.png)](https://travis-ci.org/idealista/elasticsearch_role)
```

Comment:
Could you also change the other link to .com? :)
https://travis-ci.org/idealista/elasticsearch_role -> https://travis-ci.com/idealista/elasticsearch_role

Reply:
Of course, I didn't see it.